domain/src/test/java/org/oppia/domain/classify/rules/dragAndDropSortInput/DragDropSortInputIsEqualToOrderingWithOneItemIncorrectClassifierProviderTest.kt

```diff
@@ -154,8 +158,10 @@ class DragDropSortInputIsEqualToOrderingWithOneItemIncorrectClassifierProviderTe
   }
 
   private fun setUpTestApplicationComponent() {
+    /* ktlint-disable max-line-length */
```

Comment:
Ditto. Try single line thing as mentioned in other comment.

Reply:
okay thanks